cfme/tests/automate/custom_button/test_catalog_custom_button.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from fauxfactory import gen_numeric_string
+
+from cfme.services.catalogs.catalog_items import DetailsCatalogItemView
+
+pytestmark = [pytest.mark.tier(2)]
+
+
+def test_custom_group_on_catalog_item(generic_catalog_item):
+    """
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/8h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.9
+        casecomponent: CustomButton
+        tags: custom_button
+        testSteps:
+            1. Add catalog_item
```

Comment:
Optional: You can put the setup steps in `setup`, keeping it separate from `testSteps` and also add `expectedResults` if you like.

Reply:
@valaparthvi: I don't want to increase LOC :). Its automated test. just step info if enough as per me.